workflows/microbiome/mgnify-amplicon-complete/mgnify-amplicon-pipeline-complete-tests.yml

```diff
@@ -367,34 +367,34 @@
               comment: "#"
               n: 4
     ITSoneDB_phylum_taxonomic_summary_table:
-      path: test-data/ITSoneDB_phylum_taxonomic_summary_table.tabular
-      compare: diff
-      lines_diff: 0
+      asserts:
+        - that: has_text
+          text: 'superkingdom	kingdom	phylum	DRR010481	ERR2715528'
     ITSoneDB_taxonomic_summary_table:
-      path: test-data/ITSoneDB_taxonomic_summary_table.tabular
-      compare: diff
-      lines_diff: 0
+      asserts:
+        - that: has_text
+          text: '#SampleID	DRR010481	ERR2715528'
     LSU_phylum_taxonomic_summary_table:
-      path: test-data/LSU_phylum_taxonomic_summary_table.tabular
-      compare: diff
-      lines_diff: 0
+      asserts:
+        - that: has_text
+          text: 'superkingdom	kingdom	phylum	DRR010481	ERR2715528'
```

Comment:
can we at least check for column count and/or line counts as well?

Reply:
I will add only column asserts, because i think row count might change due to the output of mapseq. Is that ok?

Reply:
https://docs.galaxyproject.org/en/latest/dev/schema.html#has-n-lines

has_n_lines can also have a min/max etc ...